pkg/k8shandler/dashboards.go

```diff
@@ -0,0 +1,45 @@
+package k8shandler
+
+import (
+	"fmt"
+	"io/ioutil"
+	"path"
+
+	"github.com/openshift/cluster-logging-operator/pkg/utils"
+)
+
+const (
+	clusterLoggingDashboardFile = "dashboards/openshift-logging-dashboard.json"
+)
+
+// CreateOrUpdateDashboards reconciles metrics dashboards component for cluster logging
+func (clusterRequest *ClusterLoggingRequest) CreateOrUpdateDashboards() (err error) {
```

Comment:
Let's use the word "ReconcileDashboards" since thats what we are doing

Reply:
The convention in reconciler.go is CreateOrUpdate<Resource> shouldn't we follow that convention? All of the other resources are CreateOrUpdate. I agree that Reconcile is probably better, but if I'm going to rename one I think we should rename them all.

Reply:
do we anticipate having additional dashboards?

Reply:
Not that I'm aware of, but potentially?